app/controllers/users_controller.rb

```diff
@@ -112,6 +111,7 @@ def show
     @phenotype_comment_replies.sort! { |b,a| a.created_at <=> b.created_at }
     @paginated_phenotype_replies = @phenotype_comment_replies
 
+    @@last_30_papers = LastUpdatedSnpsService.get_last_thirty_updated_snps
```

Comment:
This seems to be a buggy comment, the file itself is fixed and correct here in the PR